.github/workflows/ci.yml

```
@@ -14,85 +14,85 @@ jobs:
     strategy:
       matrix:
         destination: 
-          - 'platform=iOS Simulator,id=dvtdevice-DVTiOSDeviceSimulatorPlaceholder-iphonesimulator:placeholder'
+          - platform=iOS Simulator,name=iPhone 15,OS=17.0
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)